sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/ObjectAggregationIterator.scala

```diff
@@ -83,6 +85,7 @@ class ObjectAggregationIterator(
 
   override final def next(): UnsafeRow = {
     val entry = aggBufferIterator.next()
+    numOutputRows += 1
```

Comment:
This should be after the next line the same as the other `AggregationIterator`, just in case?

```scala
val res = generateOutput(entry.groupingKey, entry.aggregationBuffer)
numOutputRows += 1
res
```